File: src/fix-imports.py
import os
import re

ui_dir = './components/ui'

# Get all tsx files
for filename in os.listdir(ui_dir):
    if filename.endswith('.tsx'):
        filepath = os.path.join(ui_dir, filename)
        
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        # Remove version numbers from imports
        original_content = content
        content = re.sub(r'@radix-ui/([^@\s"\']+)@[\d.]+', r'@radix-ui/\1', content)
        content = re.sub(r'lucide-react@[\d.]+', r'lucide-react', content)
        content = re.sub(r'class-variance-authority@[\d.]+', r'class-variance-authority', content)
        
        if content != original_content:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            print(f'✅ Fixed: {filename}')
        else:
            print(f'⏭️  Skipped: {filename} (no changes needed)')

print('\n✨ All imports fixed!')

File: src/fix-imports.js
const fs = require('fs');
const path = require('path');

const uiDir = path.join(__dirname, 'components', 'ui');

// Get all .tsx files
const files = fs.readdirSync(uiDir).filter(f => f.endsWith('.tsx'));

files.forEach(file => {
  const filePath = path.join(uiDir, file);
  let content = fs.readFileSync(filePath, 'utf8');
  
  // Remove all version numbers from imports
  content = content.replace(/@radix-ui\/([^@\s"']+)@[\d.]+/g, '@radix-ui/$1');
  content = content.replace(/lucide-react@[\d.]+/g, 'lucide-react');
  content = content.replace(/class-variance-authority@[\d.]+/g, 'class-variance-authority');
  
  fs.writeFileSync(filePath, content, 'utf8');
  console.log(`✅ Fixed: ${file}`);
});

console.log('\n✨ All imports fixed!');
